### PR TITLE
Citation and licence on Overview tab

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
https://gajira.atlassian.net/browse/DSCM-441

### CODE REVIEW

**Comment by benji-glitsos-ga** [comment]
Pages:

* https://pr-167-preview.khpreview.dea.ga.gov.au/data/product/dea-coastlines/?tab=overview

File: docs/data/product/dea-coastlines/_data.yaml [hunk 1]
```diff
@@ -28,6 +28,13 @@ collection:
 doi: 10.26186/116268
 ecat: "116268"
 nci: null
+licence:
+  name: Creative Commons Attribution 4.0 International Licence
+  link: https://creativecommons.org/licenses/by/4.0/
+
+citations:
+  data_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Digital Earth Australia Coastlines. Geoscience Australia, Canberra. https://doi.org/10.26186/116268
+  paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. *Remote Sensing of Environment*, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```
Comment (robbibt) [inline docs/data/product/dea-coastlines/_data.yaml:37]:
```suggestion
  paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. Remote Sensing of Environment, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```

File: docs/data/product/dea-coastlines/_overview_2.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+**Data citation**
+
+```text
+Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Digital Earth Australia Coastlines. Geoscience Australia, Canberra. [https://doi.org/10.26186/116268](https://doi.org/10.26186/116268)
+```
+
+**Paper citation**
+
+```text
+Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. *Remote Sensing of Environment*, 267, 112734. Available: [https://doi.org/10.1016/j.rse.2021.112734](https://doi.org/10.1016/j.rse.2021.112734)
```
Comment (robbibt) [inline docs/data/product/dea-coastlines/_overview_2.md:14]:
```suggestion
Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. Remote Sensing of Environment, 267, 112734. Available: [https://doi.org/10.1016/j.rse.2021.112734](https://doi.org/10.1016/j.rse.2021.112734)
```

File: starter-kits/product/_data.yaml [hunk 3]
```diff
@@ -29,6 +29,13 @@ is_provisional: false
 # doi: 10.00000/000000
 # ecat: 100000
 # nci: a000
+licence:
+  name: Creative Commons Attribution 4.0 International Licence
+  link: https://creativecommons.org/licenses/by/4.0/
+
+# citations:
+#   data_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Digital Earth Australia Coastlines. Geoscience Australia, Canberra. https://doi.org/10.26186/116268
+#   paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. *Remote Sensing of Environment*, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```
Comment (robbibt) [inline starter-kits/product/_data.yaml:38]:
```suggestion
#   paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. Remote Sensing of Environment, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```

File: docs/data/product/dea-coastlines/_data.yaml (hunk 1 above)
Comment (benji-glitsos-ga) [reply docs/data/product/dea-coastlines/_data.yaml]: Done

File: docs/data/product/dea-coastlines/_overview_2.md (hunk 2 above)
Comment (benji-glitsos-ga) [reply docs/data/product/dea-coastlines/_overview_2.md:14]: Done

File: starter-kits/product/_data.yaml (hunk 3 above)
Comment (benji-glitsos-ga) [reply starter-kits/product/_data.yaml]: Done

**Comment by benji-glitsos-ga** [comment]
Hi @CEKrause ,

This citation + license feature PR is ready to be reviewed when you have a chance.

https://pr-167-preview.khpreview.dea.ga.gov.au/data/product/dea-coastlines/?tab=overview

I've provided two different designs on that page -- one is the original design, and the other is formatted in a table.

The Accessed date at the end of the citations is added automatically by a script. 

The citations are written as plain text (no Markdown formatting allowed) within the data YAML file:

```yaml
citations:
  data_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Digital Earth Australia Coastlines. Geoscience Australia, Canberra. https://doi.org/10.26186/116268
  paper_citation: Bishop-Taylor, R., Nanson, R., Sagar, S., Lymburner, L. (2021). Mapping Australia's dynamic coastline at mean sea level using three decades of Landsat imagery. Remote Sensing of Environment, 267, 112734. https://doi.org/10.1016/j.rse.2021.112734
```

Likewise, the license is also written in the YAML:

```yaml
licence:
  name: Creative Commons Attribution 4.0 International Licence
  link: https://creativecommons.org/licenses/by/4.0/
```

**Comment by CEKrause** [comment]
@benji-glitsos-ga I like the cite this product in the table format. I think it's much neater and sits more cleanly on the page. 
It's looking great to me!

**Comment by CEKrause** [comment]
I love the easy copy feature as well.

**Pull request merged**